### ISSUE TITLE
Map scanning undesired locations

### ISSUE THREAD

**Issue by docrockulus** [AHAAAAAAA/PokemonGo-Map#1534]
## Expected Behavior
Map scans areas defined in my startup commands

## Current Behavior
Map is scanning areas in addition to my defined scanning areas

## Possible Solution
New location stuff is probably defining new locations to scan

## Steps to Reproduce (for bugs)
1. Run map using -l switch to define a lat/long pair
2. Observe map
3. Observe areas being scanned
4. Observe areas being scanned in addition to my defined locations

## Context
I've made the map public to my city, and have many people accessing it, I suspect that they are causing this issue.

## Your Environment
Version used: 2.7
Ubuntu server 14.04
Latest commit

**Comment by AHAAAAAAA** [comment]
yes you can change the location of the map, again this release is for personal use and it was designed with that in mind.

**Comment by FallenArchangel** [comment]
A good way to disable this is to just prevent access to the #nav panel.  Just set the settings how you want, then comment it out so people cant change the location..

On the newest version as of this post, you do this by block commenting lines 44 through 106 in the /templates/map.html file.
